yadata/src/Yadata/LibAPI.hs

```diff
@@ -11,32 +11,32 @@ module Yadata.LibAPI
     createGraphForNewsletter
 ) where
 
-import           Yadata.LibCSV
```

Comment:
Avoid deleting whitespaces , once we set all on your editor this will not happen. In git you have the option to revert some hunks and keep some so you dont have to commit the whole file.